docs/blog/newsletter-august-2020.rst

```diff
@@ -0,0 +1,85 @@
+* Consider GUI clients instead of the command line. Folks mentioned GitHub Desktop, TortoiseGit (Windows), Tower, Sourcetree, and GitKraken.
+* A lot of learning Git is about building confidence with an unfamiliar workflow and new underlying concepts. Once people get through the initial learning curve, stress levels tend to settle down.
+
+---------------------
+A tale of two careers
+---------------------
+
+Want to be a technical writer and work in a different field at the same time? Is it possible to juggle two careers without burning out? The reality is that technical writing requires a lot of time and effort; it’s not easy to balance this with other work, but it’s possible.
+
+Acknowledge and prepare for the difficulties. Accept that this balancing act is hard, and that’s okay. There are only so many hours in the day. If you can’t give 100% to both, then this isn’t the right path. It’s more realistic to turn one of your jobs into a side project. Having two careers implies serious intent to progress in both things simultaneously. This will take a lot of energy, devotion, and time spent working.
+
+If you’re determined to maintain the balancing act, occasionally switching between both jobs will give you a nice change of pace. In some cases, one job may inform the other or even provide you with a fresher perspective. Also, consider how you could merge responsibilities to create a hybrid role. They may be more compatible than you think.
+
+Reflect on how to create a long-term option that fulfills most of your desires. So long as you have the physical and mental time for the two, you’ll be happy with technical writing and the other career that balances it out.
+
+-------------------------------------------
+Content reuse vs hyperlinking
+-------------------------------------------
+
+Coming soon from Claire.
+
+-------------------------------------
+What to do with tricky little details
+-------------------------------------
+
+In an ideal world, structuring documentation pages is easy. You cover the main concepts and the core tasks with clear, user-centered headings, and you’re done.
+
+But in messy reality, it can be tricky to fit all the relevant information into those headings. Where do you put non-standard uses, or fringe cases? Granular but important details that users won't find anywhere else? Questions that often come up in support?
+
+One option is to have a Frequently Asked Questions section at the bottom of the page. Readers are often familiar with FAQs as a format, and may even specifically request them. But FAQ can be a misnomer - for fringe cases, the questions are by definition not frequent. Besides, some folks feel that `FAQs are a poor content design pattern <https://gathercontent.com/blog/big-question-should-you-have-faqs-on-your-website>`__.
```

Comment:
There is an F for Fringe joke in here somewhere.

Reply:
There is, but I'm too lazy to come up with it :P